recipes/pytentiostat/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pytentiostat" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/juami/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 952e6e2743d88a3c519a9644f5514902f4b1726f67221727c71b7a37dd59ea5f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - pyfirmata
+    - pandas
+    - matplotlib
+    - pyyaml
+    - pandas
+    - numpy
+    - serial
```

Comment:
`serial` and `signal` don't appear to be available from conda-forge. How are you currently installing them?